Description: Remove bundler while running tests
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2015-12-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -1,5 +1,4 @@
 require 'rubygems'
-require 'bundler/setup'
 
 require 'gemojione'
 
